ts/packages/node/src/stores/OpenFeature.ts

```diff
@@ -0,0 +1,69 @@
+import { ConfigStore, type ConfigStoreQuery, type Config } from '@configu/ts';
+import {
+  OpenFeature,
+  type EvaluationContext,
+  type Client,
+  type Provider,
+  type EvaluationDetails,
+} from '@openfeature/server-sdk';
+
+export type OpenFeatureConfigStoreConfiguration = {
+  provider: Provider | Promise<Provider>;
+  context?: EvaluationContext;
+};
+
+export abstract class OpenFeatureConfigStore extends ConfigStore {
+  protected readonly provider: Provider | Promise<Provider>;
+  private readonly context: EvaluationContext;
+  private readonly client: Client;
+
+  protected constructor(type: string, configuration: OpenFeatureConfigStoreConfiguration) {
+    super(type);
+    const { provider, context = {} } = configuration;
+    this.provider = provider;
+    this.context = context;
```

Comment:
No need for saving the context. Set it globally on the client and let them handle the overriding and merging.

Reply:
np